src/main/java/com/example/demo/model/Employees.java

```diff
@@ -0,0 +1,124 @@
+package com.example.demo.model;
+
+import java.util.Date;
+
+public class Employees {
+
+    private String fullName;
+    private String jobTitle;
+    private String department;
+    private String businessUnit;
+    private String gender;
+    private String ethnicity;
+    private int age;
+    private Date hireDate;
+    private float annualSalary;
+    private int bonus;
```

Comment:
use sublasses of Class Number

AtomicInteger, AtomicLong, BigDecimal, BigInteger, Byte, Double, DoubleAccumulator, DoubleAdder, Float, Integer, Long, LongAccumulator, LongAdder, Short